harness/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM gcr.io/cloud-dataflow/v1beta3/python-fnapi:2.8.0
+
+# TODO(brianm): installing without version pinning might mean that the
+#   worker and driver have version mismatches. We should prefer to install
+#   the entire reqs.txt or similar.
```

Comment:
Do we need to address this before merging or is it fine for now? Would prefer to track this in a separate issue instead of a `TODO` if it's not needed for now.

Reply:
https://github.com/spotify/spotify-tensorflow/issues/97

Reply:
Awesome 🙏, can you remove this comment too?

Reply:
I kept the comment, but added the issue link inline.